python-modules/cis_hris_api/cis_hris_api/schema.py

```diff
@@ -0,0 +1,120 @@
+import json
+import graphene
+from logging import getLogger
+from cis_hris_api.common import get_table_resource
+from cis_hris_api import user
+
+logger = getLogger(__name__)
+
+
+class DotDict(dict):
```

Comment:
get that from common, instead of copying code

Reply:
Will fix in next PR.